assets/src/pages/room/RoomPage.tsx

```diff
@@ -126,7 +126,7 @@ const RoomPage: FC<Props> = ({
           {isSimulcastOn && (
             <button
               onClick={toggleSimulcastMenu}
-              className="focus:outline-none focus:shadow-outline m-1 w-full rounded bg-brand-grey-80 py-2 px-4 text-white hover:bg-brand-grey-100"
```

Comment:
Where did that go? Was that redundant?

Reply:
Yes. I've played with this using the inspect tool of Chrome. Using focus on this line didn't change anything so I've removed that. 
It's connected with upgrading tailwind to v3 since the outline classes have been vastly extended and need to be revised. I've decided that in this case the `focus:outline-none` and `focus:shadow-outline` classes are redundant.